### PR TITLE
Avoid having extra ? at the end of request url when queryItems are empty

### PR DESCRIPTION
**Pull request by tranvutuan**
Summary
----
Avoid having extra ? at the end of request url when queryItems are empty

Details
----
 
Currently `GraphQLGETTransformer` is used to create query items for GET request endpoint. If property `body` of this struct is empty ( does not have value for a given key) such as:

> key : "variables"
> value : nil

which leads to queryItems is empty. By doing `components.queryItems = queryItems`, then the url ends up to have extra ? at the end like following: `https://this_is_sample/abcd?` instead of `https://this_is_sample/abcd`

Fix
----
Only assign `queryItems` to `URLComponent` when it is not empty

### CODE REVIEW

File: Sources/Apollo/GraphQLGETTransformer.swift
Comment (designatednerd) [inline Sources/Apollo/GraphQLGETTransformer.swift:45]: note to myself to use `.apollo.isNotEmpty` here, but there are some shenanigans around the import I won't make you deal with

**Review by designatednerd — APPROVED** [review]

**Comment by designatednerd** [comment]
@tranvutuan I can't merge this until you sign the CLA, but this looks good. Mind signing it?

File: Sources/Apollo/GraphQLGETTransformer.swift
Comment (tranvutuan) [reply Sources/Apollo/GraphQLGETTransformer.swift]: I wasn't aware of `.apollo.isNotEmpty` until now. Have just updated the PR.

Comment (designatednerd) [reply Sources/Apollo/GraphQLGETTransformer.swift]: Cool - I'll have to do some silliness to make cocoapods play nice later but i appreciate you doing this bit!

**Comment by designatednerd** [comment]
Cool, thanks for the heads up.

**Comment by tranvutuan** [comment]
> Cool, thanks for the heads up.

Got a green light from our end and I have just signed the CLA.

**Comment by designatednerd** [comment]
Woo hoo, thank you!

**Pull request merged**
